packages/@uppy/tus/src/index.js

```diff
@@ -178,8 +186,9 @@ module.exports = class Tus extends Plugin {
       optsTus.onError = (err) => {
         this.uppy.log(err)
 
-        if (isNetworkError(err.originalRequest)) {
-          err = new NetworkError(err, err.originalRequest)
+        const xhr = err.originalRequest ? err.originalRequest.getUnderlyingObject() : null
```

Comment:
tus-js-client v2 introduces a HTTP abstraction layer, so the code to access XHR objects is a bit different now.